src/Http/WebApp/WebAppView.php

```diff
@@ -283,7 +327,9 @@ public function render(Request $request)
         $view = $this->view->file(__DIR__.'/../../../views/app.blade.php');
 
         $view->title = $this->buildTitle(array_get($forum, 'data.attributes.title'));
-        $view->description = $this->description;
+        $view->description = $this->description ? $this->description : array_get($forum, 'data.attributes.description');
+        $view->language = $this->language ? $this->language : $this->locales->getLocale();
+        $view->direction = $this->direction ? $this->direction : 'ltr';
```

Comment:
There is indeed a shorter version of this - see [here](http://stackoverflow.com/questions/2153180/what-is-in-php-5-3).

~~~php
$view->direction = $this->direction ?: 'ltr';
~~~